src/components/listBox/ListBoxConnected.tsx

```diff
@@ -15,6 +15,7 @@ const mapStateToProps = (state: IReactVaporState, ownProps: IListBoxOwnProps): I
     return {
         selected: list ? list.selected : [],
         active: list ? list.active : undefined,
+        items: list ? list.items : ownProps.items,

```

Comment:
I can use the same value from the store and the ownProps.

With this change, the connected and not connected can work in the same way with the same interface.

If you have more question, you can DM me for more information about this change and the mergePropsWithStatePropsPriority.

Reply:
I really like this. It makes things more seamless when the same prop may come from one place or the other.